src/components/Footer/Footer.tsx

```diff
@@ -20,32 +20,35 @@ const Footer: React.FC = () => {
         position: 'relative',
         overflow: 'hidden',
         pb: '32px',
-        backgroundColor: 'footer',
+        backgroundColor: isDark ? '#0b0b0b' : '#4d4040',
```

Comment:
we should not be hard-coding colors

Reply:
Actually, the theme color was not  working Zach has instructed  in the apeswap group to use color codes